### ISSUE TITLE
Chart editor time bar freezes game + more

### ISSUE THREAD

**Issue by SMWpalkia** [JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag#250]
### Describe your bug here. Mind you, if you are modding, please try testing your bug in a clean version of the engine instead, this ensures the bug is actually caused by the engine itself. Also, if you're using an older version of JSE, please try the latest action build in the Actions tab or the latest release build. The issue has probably been fixed since that version.

This first one may just be a me thing since I don't see everyone charting songs this long but I've been running into an issue that freezes the game if you click on the time bar in the chart editor to skip forward despite being able to manually scroll there and have notes placed there

https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/136278510/e683887b-80f9-48ff-94ba-77f6d7bf27f1

The annoying issue in psych engine where you cant type when the camera is zoomed is back

https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/136278510/8ae78598-1be2-4b1f-9393-c1dfef73faef

& finally I wasn't able to get a recording of these but I the game crashes randomly when I either A: try to rename animation names in the character editor without filling in all of them first or B: try to play a song from the chart editor shortly after opening the game (the image comes from this issue)
![image](https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/136278510/b42a53b2-3689-44ac-a1f6-dae7b49019b9)
I apologize if I it takes me a while to see responces

### Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

```bash
First one: Unsure what song length this starts at but I've noticed it happen with songs above 20 minutes and just clicking anywhere past the like 10% mark of the bar causes the issue

Second one: explains itself really

Third one: I've never payed enough attension to notice a pattern and what I thought was the issue didn't cause a crash. I saw this reported before but couldn't find an answer
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

I removed the editor music that's about it

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

Yes

**Comment by moxie-coder** [comment]
okay first of all, can you actually fucking wait for it to finish loading? just because it’s not responding doesn’t mean it’s not doing anything, secondly, update to the latest version because it looks like your on an old version

**Comment by SMWpalkia** [comment]
The only reason I didn't wait is to save time in the video. I've waited 10+ minutes prior to this and it was still frozen

And isn't there an option to check for updates? I didn't get a prompt when starting the game to update

**Comment by moxie-coder** [comment]
> The only reason I didn't wait is to save time in the video. I've waited 10+ minutes prior to this and it was still frozen
> 
> And isn't there an option to check for updates? I didn't get a prompt when starting the game to update

it was added in newer versions iirc

**Comment by JordanSantiagoYT** [comment]
> The only reason I didn't wait is to save time in the video. I've waited 10+ minutes prior to this and it was still frozen
> 
> And isn't there an option to check for updates? I didn't get a prompt when starting the game to update

yeah when removing android leftovers it broke, i just fixed it after 1.21.0
